<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createDataSenderDialogWindowFramework</class>
 <widget class="QDialog" name="createDataSenderDialogWindowFramework">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Data Sender</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="sendToCasterTab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="outputToCasterTab">
      <attribute name="title">
       <string>Output to Caster</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="2" column="2">
        <widget class="QPushButton" name="createBasestationPushButton">
         <property name="text">
          <string>Create Basestation</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>227</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>181</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Informal Name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>91</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="informalNameLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Latitude:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3" rowspan="2">
           <widget class="QLineEdit" name="latitudeLineEdit"/>
          </item>
          <item row="2" column="0" rowspan="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Longitude:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="longitudeLineEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Message Format:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QComboBox" name="messageFormatComboBox"/>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Update Frequency (1/s):</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QLineEdit" name="updateFrequencyLineEdit"/>
          </item>
          <item row="6" column="3">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="7" column="0">
           <widget class="QPushButton" name="loadSecretKeyPushButton">
            <property name="text">
             <string>Load Secret Key</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1" colspan="2">
           <widget class="QPushButton" name="loadCredentialsPushButton">
            <property name="text">
             <string>Load Credentials</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Credentials/Secret Key Required for OFFICIAL/REGISTERED basestations, but not COMMUNITY basestations. </string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="createFileDataSenderTab">
      <attribute name="title">
       <string>Output Data To File</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>403</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="filePathTextLabel">
         <property name="text">
          <string>filePath</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="selectOutputFilePushButton">
         <property name="text">
          <string>Select Output File</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="createFileDataSenderPushButton">
         <property name="text">
          <string>Create File Data Sender</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="createTCPDataReceiverTab">
      <attribute name="title">
       <string>Create TCP Server</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Publishing Port:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="createTCPDataSenderPushButton">
         <property name="text">
          <string>Create TCP Sender</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>227</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="TCPSenderPortLineEdit"/>
       </item>
       <item row="3" column="3">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="createZMQDataReceiver">
      <attribute name="title">
       <string>Create ZMQ Publisher</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="2" colspan="2">
        <widget class="QLineEdit" name="ZMQDataSenderPortLineEdit"/>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>327</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QPushButton" name="createZMQDataSenderPushButton">
         <property name="text">
          <string>Create ZMQ Sender</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Publishing Port:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
